pdc/apps/release/serializers.py

```diff
@@ -134,18 +135,29 @@ class Meta:
                   'compose_set', 'integrated_with', 'sigkey', 'allow_buildroot_push',
                   'allowed_debuginfo_services', 'allowed_push_targets')
 
+    def set_default_sigkey(self, validated_data):
+        try:
+            if hasattr(settings, 'RELEASE_DEFAULT_SIGKEY'):
+                if "sigkey" in validated_data and not validated_data["sigkey"]:
+                    validated_data["sigkey"], _ = SigKey.objects.get_or_create(key_id=settings.RELEASE_DEFAULT_SIGKEY)
+        except Exception, e:
+            raise Exception('Error(%s) when get Key_id(%s).' % (e, settings.RELEASE_DEFAULT_SIGKEY))
```

Comment:
Is the `except` block needed? What can happen here?

Reply:
You can say that about a lot of code. It actually raises different type of exception here which can change how Django reacts.

So I would rather if you remove it.

Reply:
Make sense to me.